cadence/transactions/bridge/nft/bridge_nft_to_any_cadence_address.cdc

```diff
@@ -0,0 +1,101 @@
+import "FungibleToken"
+import "NonFungibleToken"
+import "ViewResolver"
+import "MetadataViews"
+import "FlowToken"
+
+import "ScopedFTProviders"
+
+import "EVM"
+
+import "FlowEVMBridge"
+import "FlowEVMBridgeConfig"
+import "FlowEVMBridgeUtils"
+
+/// This transaction bridges an NFT from EVM to Cadence assuming it has already been onboarded to the FlowEVMBridge.
+/// Also know that the recipient Flow account must have a Receiver capable of receiving the this bridged NFT accessible
+/// via published Capability at the token's standard path.
+/// NOTE: The ERC721 must have first been onboarded to the bridge. This can be checked via the method
+///     FlowEVMBridge.evmAddressRequiresOnboarding(address: self.evmContractAddress)
+///
+/// @param nftContractAddress: The Flow account address hosting the NFT-defining Cadence contract
+/// @param nftContractName: The name of the NFT-defining Cadence contract
+/// @param id: The ERC721 id of the NFT to bridge to Cadence from EVM
+/// @param recipient: The Flow account address to receive the bridged NFT
+///
+transaction(nftContractAddress: Address, nftContractName: String, id: UInt256, recipient: Address) {
+
+    let nftType: Type
+    let receiver: &{NonFungibleToken.Receiver}
+    let scopedProvider: @ScopedFTProviders.ScopedFTProvider
+    let coa: auth(EVM.Bridge) &EVM.CadenceOwnedAccount
+    
+    prepare(signer: auth(BorrowValue, CopyValue, IssueStorageCapabilityController, PublishCapability, SaveValue, UnpublishCapability) &Account) {
+        /* --- Reference the signer's CadenceOwnedAccount --- */
+        //
+        // Borrow a reference to the signer's COA
+        self.coa = signer.storage.borrow<auth(EVM.Bridge) &EVM.CadenceOwnedAccount>(from: /storage/evm)
+            ?? panic("Could not borrow COA from provided gateway address")
+
+        // Get the ERC721 contract address for the given NFT type
+        self.nftType = FlowEVMBridgeUtils.buildCompositeType(
+                address: nftContractAddress,
+                contractName: nftContractName,
+                resourceName: "NFT"
```

Comment:
With the new token standards, NFTs aren't required to have the name `NFT` any more, so there is a small chance that NFTs might have different names in the future. Maybe this should be a transaction argument?

Reply:
Good point. I think it'd be best to update txn interfaces to take type identifier and parse the identifier for the contract name and address. Then we can just build the composite type from the identifier without the risk that the NFT has some other name.